core/common/src/main/java/alluxio/master/metastore/rocks/DataBlockIndexType.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.metastore.rocks;
+
+/**
+ * DataBlockIndexType used in conjunction with BlockBasedTable.
+ * This is a representation of {@link org.rocksdb.DataBlockIndexType}
+ * added here so core does not depend on RocksDB library.
+ * See {@link org.rocksdb.DataBlockIndexType} for more information on the values.
+ */
+public enum DataBlockIndexType {
+  kDataBlockBinarySearch,
+  kDataBlockBinaryAndHash
```

Comment:
I had to make these enums specifically for this case because that is how rocksDB uses them (they are c++ variables).

Reply:
Actually I could change them since I am converting them anyways, I was just keeping the format used in the rocks db java enums e.g. https://github.com/facebook/rocksdb/blob/main/java/src/main/java/org/rocksdb/DataBlockIndexType.java